src/core/i18n/utils.js

```diff
@@ -287,6 +295,29 @@ export function makeI18n(
       oneLineTranslationString(pluralKey), val);
   };
 
+  // Because the `addonType` we use internally is not translated, we needs this
```

Comment:
Nitpick editorialising, but this could be:

```js
// The `addonType` we use internally is not translated, so we use this
// helper to return translated add-on type names.
```